### DISCUSSION TITLE
how to fix button menu in table problem scroll down by itself

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#2054] [Q&A]
The menu button  (Menu Component) in the table has a problem.
It makes it scroll down by itself. How to fix scroll down by itself (I don't want it to slide down by itself.)

<img width="1315" alt="Screen Shot 2565-08-09 at 14 27 32" src="https://user-images.githubusercontent.com/12431734/183590907-5e2d284c-47be-48c8-b861-257f4f61f2bf.png">

https://user-images.githubusercontent.com/12431734/183591113-7fcbd5f8-66ee-42aa-af24-5819c1ab7ba0.mov

**Comment by rtivital** [comment]
https://github.com/mantinedev/mantine/issues/1982

**Reply by goodlythink** [reply]
ok thank you
transitionDuration={0}
it's work
